canonical-web-and-design/canonical.com
# Tabs refactor

## Done

- Refactored a lot of /careers, to reduce repetition and make it easier to update
- Updated tabs to rely a lot less on javascript
- Removed lorem ipsum
- Removed overview and tabs from /careers/all, it now only shows job listings

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Fill out the game and submit your choices, then click around the careers section, see that the tabs on the job categories behave as expected, and the layout of each page is not broken
- Visit /careers/all, see that it only contains job listings


## Issue / Card

Fixes #126 & #132
